test/CreateTestCert.ps1

```diff
@@ -0,0 +1,14 @@
+#
+# Generates a new test cert in a .pfx file
+# Obviously, don't actually use this to produce production certs
+#
+
+param(
+    [Parameter(Mandatory = $true)]
+    $OutFile
+)
+
+$password = ConvertTo-SecureString -Force -AsPlainText -String "password"
+$cert = New-SelfSignedCertificate -DnsName "localhost" -CertStoreLocation Cert:\CurrentUser\My\
+Export-PfxCertificate -Cert $cert -Password $password -FilePath $OutFile
+Remove-Item "Cert:\CurrentUser\My\$($cert.Thumbprint)"
```

Comment:
If you're netcoreapp-only you could use the CertificateRequest API and just make the test certs on the fly.  If you're both netfx and netcoreapp then this is a reasonable approach.

Reply:
Yeah, that's why I did this. I original wrote the tests for netcoreapp2.1 using CertificateRequest, but wanted to make sure the tests work for netfx too, so reverted to embedded a pfx file in the repo